tmt/steps/finish/__init__.py

```diff
@@ -1,12 +1,14 @@
 import copy
+from typing import Any, List, Optional
 
 import click
 import fmf
 
 import tmt
+from tmt.steps import Step
 
 
-class Finish(tmt.steps.Step):
+class Finish(Step):
```

Comment:
@guy9050 I would keep it as `tmt.steps.Step`, it is what other modules use, seems no benefit and breaks this consistency